app/javascript/components/actionable.js

```diff
@@ -48,6 +48,18 @@ export default template({
         value: this.statement.verified_on, type: 'time'
       }
 
+      if (this.page.reference_url_title) {
+        reference[wikidataClient.getPropertyID('title')] = {
+          value: this.page.reference_url_title,
+        };
+      }
+
+      if (this.page.reference_url_language) {
+        reference[wikidataClient.getPropertyID('language of work or name')] = {
+          value: this.page.reference_url_language,
```

Comment:
This object also needs a type, but in this case it's: `wikibase-entityid`. The value can't just be the string then, though, because it's a Wikidata item you're setting. After some fiddling around with imports and exports, I think these issues can be fixed with:

```diff
diff --git a/app/javascript/components/actionable.js b/app/javascript/components/actionable.js
index 971c4e0..56e63e1 100644
--- a/app/javascript/components/actionable.js
+++ b/app/javascript/components/actionable.js
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import ENV from '../env'
 import Axios from 'axios'
-import wikidataClient from '../wikiapi'
+import wikidataClient, { getItemValue } from '../wikiapi'
 import template from './actionable.html'
 import StatementChangeSummary from './statement_change_summary'
 
@@ -49,14 +49,16 @@ export default template({
       }
 
       if (this.page.reference_url_title) {
-        reference[wikidataClient.getPropertyID('title')] = {
+        references[wikidataClient.getPropertyID('title')] = {
           value: this.page.reference_url_title,
+          type: 'string',
         };
       }
 
       if (this.page.reference_url_language) {
-        reference[wikidataClient.getPropertyID('language of work or name')] = {
-          value: this.page.reference_url_language,
+        references[wikidataClient.getPropertyID('language of work or name')] = {
+          value: getItemValue(this.page.reference_url_language),
+          type: 'wikibase-entityid',
         };
       }
 
diff --git a/app/javascript/wikiapi.js b/app/javascript/wikiapi.js
index 9c5a359..9da0e34 100644
--- a/app/javascript/wikiapi.js
+++ b/app/javascript/wikiapi.js
@@ -9,7 +9,7 @@ function lowerCaseSnak(upperCase) {
   return parts[0] + '$' + parts[1].toLowerCase();
 }
 
-function getItemValue(item) {
+export function getItemValue(item) {
   return {'entity-type': 'item', 'numeric-id': Number(item.substring(1))};
 }
  
```